assets/src/dashboard/app/views/myStories/header/index.js

```diff
@@ -109,9 +106,7 @@ function Header({
         {STORY_STATUSES.map((storyStatus) => {
           const isStoryPrivate = storyStatus.status === STORY_STATUS.PRIVATE;
           const cantReadPrivate =
-            !canReadPrivatePosts ||
-            !totalStoriesByStatus.private ||
-            totalStoriesByStatus.private < 1;
+            !totalStoriesByStatus?.private || totalStoriesByStatus?.private < 1;
```

Comment:
We can probably simplify all of this like so:

```js
{STORY_STATUSES.map((storyStatus) => {
	  if (!(storyStatus.status in totalStoriesbyStatus)) {
		  return null;
	  }
	  
	  const label = storyStatus.label;
	  
	  // ...
```

No need for the special check for `private` here.

Reply:
Simpler logic. https://github.com/google/web-stories-wp/pull/8598/commits/f06114ab808bbc048f552b7367f36cebcdbadcab